rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -2,13 +2,13 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_maintain
-%global confdir foreman_maintain
+%global package_name foreman-maintain
```

Comment:
Curious why the package_name differs from the gem_name ? That seems like it can lead to confusion.